operators/src/source/gdal_source.rs

```diff
@@ -222,38 +176,58 @@ impl GdalSource {
         })
     }
 
+    pub async fn load_tile_data_async<T: gdal::raster::types::GdalType + Copy + Send + 'static>(
+        gdal_params: GdalSourceParameters,
+        time_interval: TimeInterval,
+        tile_information: TileInformation,
+    ) -> Result<RasterTile2D<T>> {
+        tokio::task::spawn_blocking(move || {
+            GdalSource::load_tile_data_impl::<T>(&gdal_params, time_interval, tile_information)
+        })
+        .await
+        .unwrap() // TODO: handle TaskJoinError
+    }
+
+    pub fn load_tile_data<T: gdal::raster::types::GdalType + Copy>(
+        &self,
+        time_interval: TimeInterval,
+        tile_information: TileInformation,
+    ) -> Result<RasterTile2D<T>> {
+        GdalSource::load_tile_data_impl(&self.gdal_params, time_interval, tile_information)
+    }
+
     ///
     /// A method to load single tiles from a GDAL dataset.
     ///
-    pub fn load_tile_data<T: gdal::raster::types::GdalType + Copy>(
-        &self,
-        time_interval: &TimeInterval,
-        tile_information: &TileInformation,
+    fn load_tile_data_impl<T: gdal::raster::types::GdalType + Copy>(
+        gdal_params: &GdalSourceParameters,
+        time_interval: TimeInterval,
+        tile_information: TileInformation,
     ) -> Result<RasterTile2D<T>> {
         // snap the time interval start instance to the dataset tick configuration
-        let time_string = self.gdal_params.tick.map(|t| {
-            t.snap_datetime(&time_interval.start().as_naive_date_time())
-                .format(&self.gdal_params.time_format)
+        let time_string = gdal_params.tick.map(|t| {
+            t.snap_datetime(&time_interval.start().as_naive_date_time().unwrap()) // TODO: replace unwrap
```

Comment:
Our default `TimeInterval`s will crash this way.

Reply:
Why?

Reply:
IF a time interval has a default range it should not use the (unfinished) ticks. If there are no ticks, it will not crash

Reply:
removed ticks from this branch